iree/turbine/kernel/wave/thread_shape_analysis.py

```diff
@@ -59,21 +80,32 @@ def handle_binaryop_conflict(custom_node: CustomOp):
     broadcast = Broadcast(broadcast_src.fx_node, dst_op.type)
     with custom_node.graph.inserting_before(custom_node.fx_node):
         broadcast.add_to_graph(custom_node.graph)
-    setattr(broadcast.fx_node, "index", dst_op.index)
-    custom_node.index = dst_op.index
     custom_node.update_arg(broadcast_idx, broadcast.fx_node)
-    return True
+    propagated_resolutions = capture_forward_slice(broadcast.fx_node, propagatable_op)
```

Comment:
Do we need to capture the forward slice? Isn't it enough to set the index on the conflicted_ops ?

Reply:
Based on our conversation, this makes sense as we need to propagate the resolved index to downstream ops.